airbyte-integrations/connector-templates/generator/plopfile.js

```diff
@@ -80,13 +80,24 @@ module.exports = function (plop) {
     }
   });
 
+
   plop.setActionType('emitSuccess', function(answers, config, plopApi){
       console.log(getSuccessMessage(answers.name, plopApi.renderString(config.outputPath, answers), config.message));
   });
 
   plop.setGenerator('Python Destination', {
     description: 'Generate a destination connector written in Python',
-    prompts: [{type:'input', name:'name', 'message': 'Connector name e.g: redis'}],
+    prompts: [
+      {type:'input', name:'name', 'message': 'Connector name e.g: redis'},
+      {
```

Comment:
This allows the user to enter whatever he would like. Do we need validation like:
* This is actually a UUID?
* The UUID does not already exist for another connector?

Reply:
> This is actually a UUID?
@maxi297 I added a validation function to check if the UUID is a valid one. 

> The UUID does not already exist for another connector?
I feel like it'd be a bit overkill to validate this in plop. But we can add a QA check in the CI to test that a definition id belongs to a single connector. (#25618)